scripts/autoharness_analyzer/src/parse_scanner_output.rs

```diff
@@ -0,0 +1,87 @@
+use anyhow::bail;
+use anyhow::Result;
+use serde::Deserialize;
+use std::collections::HashMap;
+use std::fs::File;
+use std::io::BufRead;
+use std::io::BufReader;
+use std::path::Path;
+
+// Parse the CSV files that the Kani `scanner` tool outputs
+// and store which functions are safe and unsafe
+
+// Number of columns in {crate_name}_scan_functions.csv.
+const SCANNER_COLS: usize = 6;
```

Comment:
So I guess we need to update Kani first (i.e., merge the subtree-update PR) and then this one (as the Kani version used in CI right now will have 5 columns only, and CI is failing accordingly).

Reply:
With the assumption that #354 will go in first: this PR then needs to revert https://github.com/model-checking/verify-rust-std/pull/354/commits/4fc868ebddb881fc34e14f20ad6d718fc503cfce.